modules/ROOT/pages/examples/small-scale.adoc

```diff
@@ -0,0 +1,231 @@
+
+[source,bash]
+----
+sudo useradd --system --no-create-home --shell=/sbin/nologin
+----
+
+=== Infinite Scale Data Directory
+
+Create the ocis data directory. All system data will be stored here including all files uploaded by the users.
+
+[source,bash,subs="attributes+"]
+----
+sudo mkdir -p {ocis_data}
+----
+
+Make the service user `ocis` the owner of the data directory.
+
+[source,bash,subs="attributes+"]
+----
+sudo chown ocis:ocis {ocis_data}
+----
+
+=== Infinte Scale Configuration File
+
+Create a directory and config file necessary for ocis. For security reasons, this user should have restricted permissions for this directory:
+
+[source,bash,subs="attributes+"]
+----
+sudo mkdir -p {ocis_env}
+----
+
+[source,bash,subs="attributes+"]
+----
+sudo touch {ocis_env}/ocis.env
+----
+
+[source,bash,subs="attributes+"]
+----
+sudo chown -R ocis:ocis {ocis_env}
+----
+
+Create the environment file `{ocis_env}/ocis.env` with the following content. Adjust the `OCIS_URL` variable to hold your domain where Infinite Scale will be reachable via `nginx`. Add the correct port to `OCIS_URL` if it deviates from the default port 443 like `\https://{ocis_url}:4242`. See the xref:deployment/general/general-info.adoc[General Info] section for more information about the environment variables used. 
+
+[source,bash,subs="attributes+"]
+----
+sudo vi {ocis_env}/ocis.env
+----
+
+[source,plaintext,subs="attributes+"]
+----
+OCIS_URL=https://{ocis_url}
+PROXY_HTTP_ADDR=0.0.0.0:{ocis_port}
+PROXY_TLS=false
+OCIS_INSECURE=false
+
+OCIS_LOG_LEVEL=warn
+
+OCIS_CONFIG_DIR={ocis_env}
+OCIS_BASE_DATA_PATH={ocis_data}
+----
+
+`PROXY_TLS` is set to `false` because xref:deployment/services/tls.adoc#tls-for-the-http-frontend[TLS termination] will be done by nginx.
+
+Generate the initial Infinite Scale configuration file, also see xref:deployment/general/ocis-init.adoc[The ocis init Command]:
+
+[source,bash,subs="attributes+"]
+----
+sudo -u ocis ocis init --config-path {ocis_env}
+----
+
+You will be asked if you want to configure Ininite Scale with certificate checking disabled. You can safely answer `yes`. Infinite Scale is composed of several microservices which encrypt the communication between them with TLS. By answering with `yes` the communication is still encrypted but client and server certificates aren't checked for validity. Usually this is good enough. For an extra secure deployment, an validatable TLS certificate can be provided, certificate checking can then be enabled.
+
+NOTE: The command when run successful will show you the initial admin user's password. Write it down somewhere safe so that you can log in when the setup is complete. The password can be changed in the UI later on or be reset if forgotten via xref:deployment/general/general-info.adoc#password-reset-for-the-admin-user[Password Reset for the Admin User].
```

Comment:
```suggestion
NOTE: The command when run successfully will show you the initial admin user's password. Write it down somewhere safe so that you can log in when the setup is complete. The password can be changed in the UI later on or be reset if forgotten via xref:deployment/general/general-info.adoc#password-reset-for-the-admin-user[Password Reset for the Admin User].
```